azavea/cac-tripplanner
# Add related tours to map marker popovers

## Overview

Display links to related tours in map marker popovers for destinations.


### Demo

![image](https://user-images.githubusercontent.com/960264/66439261-17fd8880-e9fd-11e9-99f1-2dab6b02c6ee.png)


### Notes

This does not show related events, only related tours.


## Testing Instructions

 * Go to 'explore' map page and click on map markers
 * The popup should list related tours, if any
 * Related tour section should be hidden if there aren't any
 * Related tours listed should link to tour detail page in a separate tab

Closes #1134.
